pact/hft.pact

```diff
@@ -452,7 +451,7 @@
     )
     (step-with-rollback
       (with-capability (SALE token seller amount timeout (pact-id))
-        (offer token seller amount timeout))
+        (offer token seller amount))
```

Comment:
btw let's change `token` to `id` for all this stuff

Reply:
also see response to your comment, instead of `sale` for caps let's use `sale-id` to not conflict with the name of the defpact